paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/SessionServiceImpl.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.server.service.impl;
+
+import org.apache.paimon.web.engine.flink.sql.gateway.client.SqlGatewayClient;
+import org.apache.paimon.web.engine.flink.sql.gateway.model.SessionEntity;
+import org.apache.paimon.web.server.data.dto.SessionDTO;
+import org.apache.paimon.web.server.service.SessionService;
+import org.apache.paimon.web.server.service.UserService;
+import org.apache.paimon.web.server.service.UserSessionManager;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+/** The implementation of {@link SessionService}. */
+@Service
+public class SessionServiceImpl implements SessionService {
+
+    private static final Integer ACTIVE_STATUS = 1;
+    private static final Integer INACTIVE_STATUS = 0;
+
+    @Autowired private UserSessionManager sessionManager;
+
+    @Autowired private UserService userService;
+
+    @Override
+    public void createSession(SessionDTO sessionDTO) {
+        try {
+            SqlGatewayClient client =
+                    new SqlGatewayClient(sessionDTO.getHost(), sessionDTO.getPort());
+            if (sessionDTO.getUid() != null) {
+                String sessionName;
+                String username = userService.getUserById(sessionDTO.getUid()).getUsername();
+                if (StringUtils.isNotBlank(sessionDTO.getName())) {
```

Comment:
Which case causes that the name of session is blank?

Reply:
The name field is unnecessary and I have removed it